src/modules/flow/converter/converter.c

```diff
@@ -2127,13 +2127,25 @@ string_to_timestamp_convert(struct sol_flow_node *node, void *data, uint16_t por
     return 0;
 }
 
-static bool
+static struct sol_blob *
 json_validate(struct sol_blob *blob, enum sol_json_type type)
 {
     struct sol_json_scanner scanner;
+    char *p;
+    size_t size;
+
+    p = (char *)blob->mem + blob->size - 1;
+    while (isspace(*p))
+        p--;
```

Comment:
Buffer underrun here: if string is comprised solely of spaces, `p--` may go past it. Ensure `p >= blob->mem` in the `while` condition.